lib/interface/cli/commands/hybrid/delete.cmd.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable max-len */
+const Command = require('../../Command');
+const runnerRoot = require('../root/runner.cmd');
+const inquirer = require('inquirer');
+const { getAllKubeContexts, getKubeContext } = require('../../helpers/kubernetes');
+const unInstallRuntime = require('../runtimeEnvironments/uninstall.cmd');
+const unInstallAgent = require('../agent/uninstall.cmd');
+const colors = require('colors');
+const DEFAULTS = require('../../defaults');
+const sdk = require('../../../../logic/sdk');
+const _ = require('lodash');
+
+const defaultNamespace = 'codefresh';
+
+const deleteCmd = new Command({
+    root: false,
+    parent: runnerRoot,
+    command: 'delete',
+    requiresAuthentication: false,
+    description: 'Deletes codefresh runner solution\'s components on kubernetes cluster',
+    webDocs: {
+        category: 'Runner',
+        title: 'Delete',
+        weight: 100,
+    },
+    // requiresAuthentication: argv => argv && !argv.token,
+    builder: yargs => yargs
+        .env('CF_ARG_') // this means that every process.env.CF_ARG_* will be passed to argv
+        .option('name', {
+            describe: 'Agent\'s name to be deleted',
+        })
+        .option('url', {
+            describe: 'Codefresh system custom url',
+            default: DEFAULTS.URL,
+        })
+        .option('kube-context-name', {
+            describe: 'Name of the kubernetes context on which venona should be installed [$CF_ARG_KUBE_CONTEXT_NAME]',
+        })
+        .option('kube-namespace', {
+            describe: 'Name of the namespace on which venona should be installed [$CF_ARG_KUBE_NAMESPACE]',
+        })
+        .option('kube-config-path', {
+            describe: 'Path to kubeconfig file (default is $HOME/.kube/config)',
+        })
+        .option('verbose', {
+            describe: 'Print logs',
+        }),
+    handler: async (argv) => {
+        const {
+            'kube-config-path': kubeConfigPath,
+        } = argv;
+        let {
+            'kube-context-name': kubeContextName,
+            'kube-namespace': kubeNamespace,
+            name: agentName,
+        } = argv;
+
+        const questions = [];
+        if (!kubeContextName) {
+            const contexts = getAllKubeContexts(kubeConfigPath);
+            const currentKubeContext = getKubeContext(kubeConfigPath);
+
+            questions.push({
+                type: 'list',
+                name: 'context',
+                message: 'Name of Kubernetes context to use',
+                default: currentKubeContext,
+                choices: contexts,
+            });
+        }
+        if (!kubeNamespace) {
+            questions.push({
+                type: 'input',
+                name: 'namespace',
+                default: defaultNamespace,
+                message: 'Kubernetes namespace to remove Runner components from it ',
```

Comment:
I think you should change this to: 'Kubernetes namespace to remove Codefresh Runner components from'